docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -89,54 +90,68 @@ OpenSearch Python client offers a helper called bulk() which allows us to send m
 
-First, export the ``SERVICE_URI`` variable with your OpenSearch service URI address and index name from the previous script:
+Get data mapping with Python
+----------------------------
+When we sent the data to OpenSearch cluster on :ref:`load the data with Python <load-data-with-python>`, no data structure was specified.
+This is possible because OpenSearch uses **dynamic mapping** to automatically add the fields to their type mapping. 
 
-.. code:: bash
+Use ``get_mapping`` method with the index name as the argument to check the mapping definition of your data.
 
-    export SERVICE_URI="YOUR_SERVICE_URI_HERE/epicurious-recipes"
+.. code:: python
 
-1. Execute a basic search for the word ``vegan`` across all documents and fields:
+    from pprint import pprint
 
-.. code:: bash
+    INDEX_NAME = 'epicurious-recipes'
+    mapping_data = os_client.indices.get_mapping(INDEX_NAME)
 
-    http "$SERVICE_URI/_search?q=vegan"
+    # Find index doc_type
+    doc_type = list(mapping_data[INDEX_NAME]["mappings"].keys())[0]
 
-2. Search for ``vegan`` in the ``desc`` or ``title`` fields only: 
+    schema = mapping_data[INDEX_NAME]["mappings"][doc_type]
+    print(f"Fields: {list(schema.keys())} \n")
+    pprint(schema, width=50, indent=1)
 
-.. code:: bash
+You should be able to see this output:
 
-    http POST "$SERVICE_URI/_search" <<< '
-    {
-        "query": {
-            "multi_match": {
-                "query": "vegan",
-                "fields": ["desc", "title"]
-            }
-        }
-    }
-    '
+.. code-block:: bash
 
-3. Search for recipes published only in 2013:
+    Fields: ['calories', 'categories', 'date', 'desc', 'directions', 'fat', 'ingredients', 'protein', 'rating', 'sodium', 'title'] 
 
-.. code:: bash
+And the mapping with the fields and their respective types.
+
+.. code-block:: bash
+
+    {'calories': {'type': 'float'},
+    'categories': {'fields': {'keyword': {'ignore_above': 256,
+                                        'type': 'keyword'}},
+                    'type': 'text'},
+    'date': {'type': 'date'},
+    'desc': {'fields': {'keyword': {'ignore_above': 256,
+                                    'type': 'keyword'}},
+            'type': 'text'},
+    'directions': {'fields': {'keyword': {'ignore_above': 256,
+                                        'type': 'keyword'}},
+                    'type': 'text'},
+    'fat': {'type': 'float'},
+    'ingredients': {'fields': {'keyword': {'ignore_above': 256,
+                                            'type': 'keyword'}},
+                    'type': 'text'},
+    'protein': {'type': 'float'},
+    'rating': {'type': 'float'},
+    'sodium': {'type': 'float'},
+    'title': {'fields': {'keyword': {'ignore_above': 256,
+                                    'type': 'keyword'}},
+            'type': 'text'}}
+
+
+
+.. seealso::
+
+    More about `OpenSearch mapping <https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/create-index/#mappings>`_ 
```

Comment:
Let's make it a whole sentence - "Read more about in the official OpenSearch Documentation", so that the readers are aware where the links brings them